bin/plugin/commands/performance.js

```diff
@@ -30,54 +30,57 @@ const config = require( '../config' );
 /**
  * @typedef WPRawPerformanceResults
  *
- * @property {number[]} timeToFirstByte      Represents the time since the browser started the request until it received a response.
```

Comment:
This just adds one new properties: largestContentfulPaint. The rest is indentation.